spec/factories/payments.rb

```diff
@@ -2,6 +2,6 @@
 # frozen_string_literal: true
 FactoryGirl.define do
   factory :payment do
-    amount { FactoryHelper::MySQL.integer }
+    amount { FactoryHelper::MySQL.integer(unsigned: true) }
```

Comment:
`unsigned int` goes to 4294967295 which will overflow.  Try `FactoryHelper::MySQL.int(min: 0, max: 2147483647)`